test-app/config/ember-try.js

```diff
@@ -20,6 +20,24 @@ module.exports = async function () {
         npm: {
           devDependencies: {
             'ember-source': '~5.12',
+            'ember-load-initializers': '^3.0.0', // v3 needed for ember 5+
+            'ember-data': '~5.3', // couldnt get earlier versions to work with ember-source 5.12
```

Comment:
technically ember-data 4.12 should support ember-source 5.x but I could not get past this error until I bumped ember-data to 5.x
```
Global error: Uncaught TypeError: Cannot set property default of #<Object> which has only a getter
```